config/samples/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 ## Append samples you want in your CSV to this file as resources ##
 resources:
 - ocs_v1alpha1_managedocs.yaml
```

Comment:
Leftovers from the last PR? if so, let's remove it

Reply:
I think this should be part of a separate PR as there are a few more leftovers.

Reply:
Ok, please open an issue to clean up these leftovers, specifying the different location where cleanup is needed

Reply:
https://github.com/red-hat-storage/managed-fusion-agent/issues/10